DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -1,4 +1,141 @@
+	"HLT_Mu20_v*","HLT_TkMu20_v*",
+	"HLT_IsoMu24_eta2p1_v*",
+	"HLT_IsoMu24_v*",
+	"HLT_IsoMu27_v*",
+	"HLT_IsoMu20_v*",
+	"HLT_IsoTkMu24_eta2p1_v*",
+	"HLT_IsoTkMu24_v*",
+	"HLT_IsoTkMu27_v*",
+	"HLT_IsoTkMu20_v*"
+)
+
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_eleref = hltHIGmonitoring.clone()
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_eleref.nmuons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_eleref.nelectrons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_eleref.FolderName = cms.string('HLT/Higgs/DiLepton/HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_eleref/')
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_eleref.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_v*")
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_eleref.denGenericTriggerEventPSet.hltPaths = cms.vstring(
+        "HLT_Ele27_WPTight_Gsf_v*",
+	"HLT_Ele35_WPTight_Gsf_v*"
+)
+
+#####HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v#####
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_muref = hltHIGmonitoring.clone()
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_muref.nmuons = cms.uint32(1)
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_muref.nelectrons = cms.uint32(1)
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_muref.FolderName = cms.string('HLT/Higgs/DiLepton/HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_muref/')
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_muref.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v*") # 
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_muref.denGenericTriggerEventPSet.hltPaths = cms.vstring(
+	"HLT_Mu20_v*",
+	"HLT_TkMu20_v*",
+	"HLT_IsoMu24_eta2p1_v*",
+	"HLT_IsoMu24_v*",
+	"HLT_IsoMu27_v*",
+	"HLT_IsoMu20_v*",
+	"HLT_IsoTkMu24_eta2p1_v*",
+	"HLT_IsoTkMu24_v*",
+	"HLT_IsoTkMu27_v*",
+	"HLT_IsoTkMu20_v*"
+)
+
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_eleref = hltHIGmonitoring.clone()
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_eleref.nmuons = cms.uint32(1)
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_eleref.nelectrons = cms.uint32(1)
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_eleref.FolderName = cms.string('HLT/Higgs/DiLepton/HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_eleref/')
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_eleref.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_TrkIsoVVL_Ele23_CaloIdL_TrackIdL_IsoVL_v*")
+mu8TrkIsoVVLEle23CaloIdLTrackIdLIsoVL_eleref.denGenericTriggerEventPSet.hltPaths = cms.vstring(
+        "HLT_Ele27_WPTight_Gsf_v*",
+	"HLT_Ele35_WPTight_Gsf_v*"
+)
+
+###############################same flavour trilepton monitor####################################
+########TripleMuon########
+higgsTrimumon = hltHIGmonitoring.clone()
+higgsTrimumon.FolderName = cms.string('HLT/Higgs/TriLepton/HLT_TripleMu_12_10_5/')
+higgsTrimumon.nmuons = cms.uint32(3)
+higgsTrimumon.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_TripleMu_12_10_5_v*") # 
+higgsTrimumon.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_v*","HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_v*")
```

Comment:
These paths are in https://its.cern.ch/jira/browse/CMSHLT-1311 and https://its.cern.ch/jira/browse/CMSHLT-1250.